### ISSUE TITLE
task is not valid under any of the given schemas (schema[tasks])

### ISSUE THREAD

**Issue by nodiscc** [ansible/schemas#364]
ansible-lint throws this error:

```
schema: {'name': 'send test netdata notification', 'become': True, 'become_user': 'netdata', 'command': '/usr/libexec/netdata/plugins.d/alarm-notify.sh test', 'environment': {'NETDATA_ALARM_NOTIFY_DEBUG': 1}, 'changed_when': True} is not valid under any of the given schemas (schema[tasks])
roles/monitoring_netdata/tasks/utils-netdata-test-notifications.yml:1  Returned errors will not include exact line numbers, but they will mention
the schema name being used as a tag, like ``playbook-schema``,
``tasks-schema``.

This rule is not skippable and stops further processing of the file.

Schema bugs should be reported towards (https://github.com/ansible/schemas) project instead of ansible-lint.

If incorrect schema was picked, you might want to either:

* move the file to standard location, so its file is detected correctly.
* use ``kinds:`` option in linter config to help it pick correct file type.
```

I can't find why this error is triggered, as the task definition looks valid, and running the task appears works as expected:

```yaml
- name: send test netdata notification
  become: yes
  become_user: netdata
  command: /usr/libexec/netdata/plugins.d/alarm-notify.sh test
  environment:
    NETDATA_ALARM_NOTIFY_DEBUG: 1
  changed_when: True
```

Is this a bug in the tasks schema? Thanks

**Comment by apatard** [comment]
You have 2 problems:
- ``become: yes``: I've got it in the past: "yes" is not a valid YAML value, use "True". See README.md for a little bit more information
- ``NETDATA_ALARM_NOTIFY_DEBUG: 1``: I understand It's not obvious but the issue is with the environment variable value. As environment variables are string, the schema looks for something like ``<string>: <string>`` and ``1`` is a number. Using ``"1"`` instead will solve this.
If you have ``check-jsonschema`` at hand, using something like:
``check-jsonschema -v --schemafile /path/to/ansible-tasks.json /path/to/mytaskfile.yml``
will help (take the ansible-tasks.json from the git repo for instance).

To be complete, this will pass:
```
- name: send test netdata notification
  become: True
  become_user: netdata
  command: /usr/libexec/netdata/plugins.d/alarm-notify.sh test
  environment:
    NETDATA_ALARM_NOTIFY_DEBUG: "1"
  changed_when: True
```

**Comment by nodiscc** [comment]
Setting `NETDATA_ALARM_NOTIFY_DEBUG: "1"` was enough to fix the problem. Thank you
